ui/app/routes/variables.js

```diff
@@ -13,9 +14,11 @@ export default class VariablesRoute extends Route.extend(WithForbiddenState) {
       this.router.transitionTo('/jobs');
     }
   }
-  model() {
+  async model() {
+    const variables = await this.store.findAll('variable');
     return RSVP.hash({
-      variables: this.store.findAll('variable'),
+      variables,
+      pathTree: new PathTree(variables),
```

Comment:
the /variables route now returns both a PathTree (traversal structure) and the variables themselves (with all metadata and info, like time created). 

There is a design decision that could be made to combine these here, but it seems worthwhile to keep them separate.